src/deploy/hosting/convertConfig.ts

```diff
@@ -57,27 +60,99 @@ export async function convertConfig(
     return out;
   }
 
-  const endpointBeingDeployed = (serviceId: string, region: string = "us-central1") => {
+  const endpointFromBackend = (
+    targetBackend: backend.Backend,
+    serviceId: string,
+    platform?: string,
+    region?: string
+  ): backend.Endpoint | undefined => {
+    const backendsForId = backend.allEndpoints(targetBackend).filter((endpoint) => {
+      return endpoint.id === serviceId;
+    });
+
+    const matchingBackends = backendsForId.filter((endpoint) => {
+      return (
+        (!region || endpoint.region === region) && (!platform || endpoint.platform === platform)
+      );
+    });
+
+    if (matchingBackends.length > 1) {
+      throw new FirebaseError(`More than one backend found for function name: ${serviceId}.
+      If the function is deployed in multiple regions, you must specify a region.`);
+    }
+
+    if (matchingBackends.length === 1) {
+      const endpoint = matchingBackends[0];
+      if (endpoint && isHttpsTriggered(endpoint)) {
+        return endpoint;
+      }
+    }
+    return undefined;
```

Comment:
```suggestion
    return;
```

`return` with no parameter will return `undefined`